package.json

```diff
@@ -20,7 +20,7 @@
     "cy:open": "cypress open"
   },
   "engines": {
-    "node": "16.18.1"
+    "node": "20"
```

Comment:
`"node": ">=20.0.0"` - else will echo bad engine error all the time :)

Reply:
Good catch!

Reply:
Resolved in https://github.com/Scottish-Natural-Heritage/cross-service-pages/pull/39/commits/d8e9f92b9786f531f2dfcc7768febad2b3bd6b7d.